ndk-make.sh

```diff
@@ -63,12 +52,73 @@ mkdir -p armeabi-v7a
 mkdir -p arm64-v8a
 mkdir -p x86
 mkdir -p x86_64
-cp deltachat-core-rust/target/armv7-linux-androideabi/release/libdeltachat.a armeabi-v7a
-cp deltachat-core-rust/target/aarch64-linux-android/release/libdeltachat.a arm64-v8a
-cp deltachat-core-rust/target/i686-linux-android/release/libdeltachat.a x86
-cp deltachat-core-rust/target/x86_64-linux-android/release/libdeltachat.a x86_64
+
+cd deltachat-core-rust
+
+# fix build on MacOS Catalina
+unset CPATH
+
+if test -z $1; then
+  # Full build
```

Comment:
```suggestion
  echo "Full build."
```

it is good to know what is going on :)